app/build.gradle

```diff
@@ -45,13 +46,16 @@ android {
 
 dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
-    implementation platform('org.jetbrains.kotlin:kotlin-bom:1.8.0')
```

Comment:
no es algo que necesites hacer para este PR, pero te recomiendo que veas esto: https://developer.android.com/build/migrate-to-catalogs para el manejo de dependencias